importer/tasks.py

```diff
@@ -239,14 +240,33 @@ def get_item_info_from_result(result):
     return m.group(1), item_url
 
 
-async def import_item_count_from_url(import_url):
+def fetch_all_urls(items):
+    with concurrent.futures.ThreadPoolExecutor(max_workers=15) as executor:
+        result = executor.map(import_item_count_from_url, items)
+    finals = []
+    totals = 0
+
+    for value, score in result:
+        totals = totals + score
+        finals.append(value)
+
+    return finals, totals
+
+
+def import_item_count_from_url(import_url):
     """
     Given a loc.gov URL, return count of files from the resources section
     """
-    resp = requests.get(import_url, params={"fo": "json"})
-    resp.raise_for_status()
-    item_data = resp.json()
-    return len(item_data["resources"][0]["files"])
+    try:
+
+        resp = requests.get(import_url, params={"fo": "json"})
+        resp.raise_for_status()
+        item_data = resp.json()
+        output = len(item_data["resources"][0]["files"])
+        return f"{import_url} {output}", output
+
+    except Exception as exc:
+        return f"{exc}", 0
```

Comment:
This exception should be logged to Sentry to save time when you're asked to troubleshoot:

```python
logging.warning("Unable to retrieve file count from %s: %s", import_url, exc)
```